repository/z.json

```
@@ -158,6 +158,16 @@
 				}
 			]
 		},
+		{
+			"name": "ZigUnofficial",
+			"details": "https://github.com/aidenfoxivey/sublime-zig-unofficial",
+			"releases": [
+				{
+					"sublime_text": ">=3000",
```

Please change this to 3092, as per the bot review. I doubt we have people using versions between 3000 and 3092, but it's a very common build dependency that helps set expectations and it's also the technically more correct way.
```suggestion
					"sublime_text": ">=3092",
```